### PR TITLE
Add read torque method to joint

### PR DESCRIPTION
**Pull request by Ishadijcks**
And make sure it's published by the `joint_state_controller`

### CODE REVIEW

**Review by JitskedeVries — APPROVED** [review]
Looks good

File: march_hardware/src/IMotionCube.cpp [hunk 1]
```diff
@@ -155,6 +155,15 @@ float IMotionCube::getAngleRad()
   return this->encoder.getAngleRad(this->misoByteOffsets[IMCObjectName::ActualPosition]);
 }
 
+float IMotionCube::getTorque()
+{
+  ROS_ASSERT_MSG(this->misoByteOffsets.count(IMCObjectName::ActualTorque) == 1, "ActualTorque not defined in PDO "
+                                                                                "mapping, so can't get angle");
```
Comment (martijnvandermarel) [inline march_hardware/src/IMotionCube.cpp:161]: Change to "can't get torque"

**Review by martijnvandermarel — COMMENTED** [review]
@JitskedeVries @Ishadijcks This print is wrong

File: march_hardware/src/IMotionCube.cpp (hunk 1 above)
Comment (Ishadijcks) [reply march_hardware/src/IMotionCube.cpp:161]: https://github.com/project-march/hardware-interface/pull/130

**Pull request merged**